spec/event_builder_spec.rb

```diff
@@ -253,6 +253,18 @@
     expect(conversion_event.http_verb).to eq(:post)
   end
 
+  it 'should create_conversion_event when Event is used in multiple experiments' do
+    @expected_conversion_params[:visitors][0][:snapshots][0][:decisions].push(
```

Comment:
The setup of this test is not very clear from just looking at this. Can you split this out into its own scenario with its own setup? Specifically, the scenario testing event tracking with multiple experiments.